modules/jolt_physics/objects/jolt_object_3d.h

```diff
@@ -66,6 +66,7 @@ class JoltObject3D {
 	RID rid;
 	ObjectID instance_id;
 	JoltSpace3D *space = nullptr;
+	JPH::Body *jolt_body = nullptr;
 	JPH::BodyID jolt_id;
```

Comment:
Change looks good. Only remark I have is that `jolt_id` is now used in so few places that you might as well remove it and replace uses by `jolt_body->GetID()`.

Reply:
That's a good point. I'll remove it.